Neos.Neos/Classes/Controller/Service/NodesController.php

```diff
@@ -107,19 +110,26 @@ class NodesController extends ActionController
     public function indexAction(
         string $searchTerm = '',
         array $nodeIds = [],
+        array $nodeIdentifiers = [],
         string $workspaceName = 'live',
         array $dimensions = [],
         array $nodeTypes = ['Neos.Neos:Document'],
         string $contextNode = null
     ): void {
+        $nodeIds = $nodeIds ?: $nodeIdentifiers;
         $contentRepositoryId = SiteDetectionResult::fromRequest($this->request->getHttpRequest())
             ->contentRepositoryId;
         $contentRepository = $this->contentRepositoryRegistry->get($contentRepositoryId);
 
-
-        $nodeAddress = $contextNode
-            ? NodeAddressFactory::create($contentRepository)->createFromUriString($contextNode)
+        $nodePath = $contextNode
+            ? AbsoluteNodePath::tryFromString($contextNode)
             : null;
+        $nodeAddress = null;
+        if (!$nodePath) {
+            $nodeAddress = $contextNode
+                ? NodeAddressFactory::create($contentRepository)->createFromUriString($contextNode)
+                : null;
+        }
```

Comment:
I have a hard time trying to understand what this code does.. Couldn't this be simplified to s.th. like:
```suggestion
        $nodeAddress = null;
        if ($contextNode !== null && AbsoluteNodePath::isAbsolutePath($contextNode)) {
            $nodeAddress = NodeAddressFactory::create($contentRepository)->createFromUriString($contextNode);
        }
```